.openpublishing.redirection.json

```diff
@@ -3544,9 +3544,13 @@
       "redirect_url": "/dotnet/visual-basic/developing-apps/windows-forms/index",
       "redirect_document_id": true
     },
+    {
+      "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio.md",
+      "redirect_url": "https://docs.microsoft.com/visualstudio/porting/porting-migrating-and-upgrading-visual-studio-projects"
+    },
     {
       "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md",
-      "redirect_url": "/dotnet/visual-basic/getting-started/breaking-changes-in-visual-studio"
+      "redirect_url": "https://docs.microsoft.com/visualstudio/porting/porting-migrating-and-upgrading-visual-studio-projects"
     },
```

Comment:
I've updated this redirection to avoid double redirect. Is that correct ? or it should be kept and only the new one added ?

Reply:
It should be updated, but you need to leave ?view=vs-2015 on the end of the link for both redirect URLs. Otherwise, it'll go to the VS 2019 version of that topic, which doesn't apply in this case.